docs/serving/samples/hello-world/helloworld-php/README.md

```diff
@@ -57,6 +57,14 @@ cd knative-docs/serving/samples/hello-world/helloworld-php
    RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
    ```
 
+1. Create a `.dockerignore` file to ensure that any files related to a local build, do not affect the container that you build for deployment.
```

Comment:
```suggestion
1. Create a `.dockerignore` file to ensure that any files related to a local build do not affect the container that you build for deployment.
```

I think this comma is extraneous, though I defer to Richie on language choices.

Reply:
nice catch Evan (I shouldn't have left that in there)!